From b010d04f7d366e98269b53c88006627f20c1be3a Mon Sep 17 00:00:00 2001
From: mohit <mohit.p@variscite.com>
Date: Fri, 12 Nov 2021 10:17:13 -0800
Subject: [PATCH 3/3] imx8mm: Remove uart2 and uart4 domain restrictions

i.MX8MM ATF code assigns UART2 to the A53 Domain and UART4 to the M4 domain.
Remove these assignments to allow for usage in either domain.
---
 plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
index b5733513b..31c060e0a 100644
--- a/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mm/imx8mm_bl31_setup.c
@@ -61,9 +61,6 @@ static const struct imx_rdc_cfg rdc[] = {
 	RDC_MDAn(RDC_MDA_M4, DID1),
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART4, D1R | D1W),
-	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
-	RDC_PDAPn(RDC_PDAP_UART1, D0R | D0W),
 
 	/* memory region */
 
-- 
2.25.1

